include/image-commands.mk

```diff
@@ -316,6 +316,14 @@ define Build/gzip
 	@mv $@.new $@
 endef
 
+define Build/gzip-filename
+	@mkdir -p $@.tmp
+	@cp $@ $@.tmp/$(word 1,$(1))
+	$(STAGING_DIR_HOST)/bin/gzip -f -9 -N -c $@.tmp/$(word 1,$(1)) $(word 2,$(1)) > $@.new
```

Comment:
You call this function with `FP221B-9.99-AP-build999-999999-patch99` in your build. What is `$(word 2,$(1)) ` doing here?

Reply:
I copied the pattern literally from `Build/gzip`, since it is intended to be a generic build recipe for multiple use cases just like that one

Reply:
ok fine.
This is then used to provide extra paramaters for gzip.